lonesha256.h

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Add CC0 to Readme#License?

Reply:
No need per the terms of the license, but up to you. Well, since you already mention public domain libs, I guess it makes sense.